salalib/agents/agentprogram.cpp

```diff
@@ -0,0 +1,356 @@
+// sala - a component of the depthmapX - spatial network analysis platform
+// Copyright (C) 2000-2010, University College London, Alasdair Turner
+// Copyright (C) 2011-2012, Tasos Varoudis
+// Copyright (C) 2019, Petros Koutsolampros
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+#include "agentprogram.h"
+
+#include "genlib/pafmath.h"
+#include "genlib/stringutils.h"
+
+#include <fstream>
+
+AgentProgram::AgentProgram() {
+    m_sel_type = SEL_LOS;
+    m_steps = 3;
+    m_vbin = 7;
+    m_destination_directed = false;
+    m_los_sqrd = false;
+}
+
+/*
```

Comment:
Could we just purge commented out code? It's still there in the github history...